styles.css

```diff
@@ -90,6 +95,362 @@ ul {
+  background-repeat: no-repeat;
+  background-size: 327px, 386px;
+  background-position: center;
+  height: 470px;
+}
+
+.other-projects > button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  position: auto;
+  color: #fff;
+  width: 327px;
+  height: 48px;
+  border: none;
+  background: #ff6b00;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0;
+}
+
+.other-projects > ul > li {
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 10px 12px 10px 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  margin-right: 12px;
+}
+
+.other-projects > ul {
+  margin-bottom: 22px;
+  margin-left: 40px;
+  margin-top: 0;
+  padding: 0;
+}
+
+.other-projects > .title-post {
+  margin-bottom: 12px;
+  margin-left: 40px;
+  text-align: left;
+  margin-top: 140px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.other-projects > .supporting-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 12px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #fff;
+}
+
+/* ----------------- About Me ---------------- */
+.about-me {
+  background: #1c1a19;
+  background-image: url(Assets/about-me-bg.svg);
+  background-repeat: no-repeat;
+  height: 2150px;
+  background-position-y: 25%;
+  background-position-x: right;
+}
+
+.about-me > .primary-text {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 120px;
+  margin-bottom: 24px;
+  text-align: center;
+}
+
+.about-me > .description {
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 24px;
+  margin-top: 0;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.about-me > button {
+  width: 161px;
+  height: 48px;
+  margin-left: 24px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+  color: #fff;
+}
+
+.about-me > hr {
+  border: 1px solid #fff;
+  border-top: none;
+  margin-top: 230px;
+  margin-bottom: 136px;
+  width: 327px;
+}
+
+.about-me > h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #fff;
+}
+
+.about-me > h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  margin-bottom: 32px;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+}
```

Comment:
- Please kindly remember to update this part of your CSS to suite the technologies and categories `list tags` that you would have updated. 🌸 🌸